README.md

```diff
@@ -1,112 +1,234 @@
 # WhisperBot
-Welcome to WhisperBot. WhisperBot builds upon the capabilities of the [WhisperLive](https://github.com/collabora/WhisperLive) and [WhisperSpeech](https://github.com/collabora/WhisperSpeech) by integrating Mistral, a Large Language Model (LLM), on top of the real-time speech-to-text pipeline. WhisperLive relies on OpenAI Whisper, a powerful automatic speech recognition (ASR) system. Both Mistral and Whisper are optimized to run efficiently as TensorRT engines, maximizing performance and real-time processing capabilities.
+
+
+Welcome to WhisperBot. WhisperBot builds upon the capabilities of the
```

Comment:
I can open another PR, to replace WhisperBot with WhisperFusion.

Reply:
Ah, sorry. I tried to include all intermediate changes but missed the obvious one.